download/release-notes-1.1.0.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Release Notes 1.1.0
+sidebar_position: 10
+​--- 
```

Comment:
the latest version release-notes sidebar_position  should be sorted first .
Maybe It is better to add the wip description before the official release 
```
title: Release Notes 1.1.0(WIP)
sidebar_position: 8
```